src/components/dropdownMenu.js

```diff
@@ -0,0 +1,208 @@
+import React, { useState, useRef, useEffect } from 'react';
+import styled from '@emotion/styled';
+
+import { Box, css } from 'theme-ui';
+import { InlineBlock } from './layout';
+import { PropTypes } from 'prop-types';
+import {
+  applyVariation,
+  UP_ARROW,
+  DOWN_ARROW,
+  ENTER_KEY,
+  TAB_KEY,
+} from './../utils/getStyles'; // TODO: move to separate file after pills merged
+
+const DropdownContainer = styled(InlineBlock)({
+  position: 'relative',
+});
+
+const DropDown = styled(InlineBlock)(
+  ({ theme, variant }) => applyVariation(theme, `${variant}.dropdownTrigger`, 'dropdownMenu')
+);
+
+const MenusContainer = styled(Box)`
+  ${({ theme, displayTop }) => css({
+    position: 'absolute',
+    width: '200px',
+    background: 'white',
+    padding: '10px 0',
+    borderStyle: 'solid',
+    borderWidth: '1px',
+    borderColor: 'borderGray',
+    borderRadius: '6px',
+    bottom: displayTop,
+  })(theme)}
+  ${({theme, variant}) => applyVariation(theme, `${variant}.dropdownContainer`, 'dropdownMenu')}
+`;
+
+const Menu = styled(Box)`
+  ${({ theme }) => css({
+    position: 'relative',
+    padding: '5px 10px',
+    '&:hover': {
+      background: 'borderGray',
+    },
+  })(theme)}
+  ${({ theme, variant }) => applyVariation(theme, `${variant}.dropdownOption`, 'dropdownMenu')}
+  ${({ theme, variant, hover }) => hover ? applyVariation(theme, `${variant}.dropdownOption.&:hover`, 'dropdownMenu') : ''}
+  ${({ theme, variant, hover }) => hover ? applyVariation(theme, `${variant}.dropdownOption.&:focus`, 'dropdownMenu') : ''}
```

Comment:
- [ ] Not sure why we need this

Reply:
In some case the mouse-over style and keyboard focus style can be different, initially hover style is applied when the focus (keyboard) style is not given